ui/src/components/BaseFormView/BaseFormView.test.tsx

```diff
@@ -66,3 +73,63 @@ it('should pass default values to custom component correctly', async () => {
 
     expect((customModal as HTMLSelectElement)?.value).toEqual('input_three');
 });
+
+it.each([
+    { page: 'configuration', config: getGlobalConfigMockGroupsForConfigPage(), service: 'account' },
+    { page: 'inputs', config: getGlobalConfigMockGroupsFoInputPage(), service: 'demo_input' },
+])('entities grouping for page works properly %s', async ({ config, page, service }) => {
+    setUnifiedConfig(config);
+    const getElementsByGroup = (group: string) => {
+        const firstField = screen.queryByText(`Text 1 Group ${group}`);
+        const secondField = screen.queryByText(`Text 2 Group ${group}`);
+        return { firstField, secondField };
+    };
+    const verfyDisplayedElement = (group: string) => {
+        const { firstField, secondField } = getElementsByGroup(group);
+        expect(firstField).toBeInTheDocument();
+        expect(secondField).toBeInTheDocument();
+    };
+    const verifyNotDisplayedElement = (group: string) => {
+        const { firstField, secondField } = getElementsByGroup(group);
+        expect(firstField).not.toBeInTheDocument();
+        expect(secondField).not.toBeInTheDocument();
+    };
+    render(
+        <BaseFormView
+            page={page as StandardPages}
+            stanzaName={STANZA_NAME}
+            serviceName={service}
+            mode="create"
+            currentServiceState={{}}
+            handleFormSubmit={handleFormSubmit}
+        />
+    );
+    const group1Header = await screen.findByText('Group 1', { exact: true });
+    expect(group1Header).toBeInTheDocument();
```

Comment:
```suggestion
    expect(group1Header).toBeInTheDocument();
```

this assertion always passes (as well as similar assertions after findBy*.
<img width="500" alt="image" src="https://github.com/user-attachments/assets/ca09a77a-f0b5-4820-815e-cff10ea5c07a">

https://testing-library.com/docs/react-testing-library/cheatsheet

Reply:
true